src/Parser.js

```diff
@@ -764,52 +706,60 @@ export default class Parser {
     }
 
     /**
-     * If `optional` is false or absent, this parses an ordinary group,
-     * which is either a single nucleus (like "x") or an expression
-     * in braces (like "{x+y}") or an implicit group, a group that starts
-     * at the current position, and ends right before a higher explicit
+     * Parses an argument.
+     */
```

Comment:
Are arguments the only groups where we specify a mode?  If so, it might be helpful to add a comment saying such.